app/models/social-link.js

```diff
@@ -4,14 +4,15 @@ import attr from 'ember-data/attr';
 import ModelBase from 'open-event-frontend/models/base';
 import { belongsTo } from 'ember-data/relationships';
 import { computedSegmentedLink, socialPlatforms } from 'open-event-frontend/utils/computed-helpers';
+import { capitalize } from 'lodash-es';
```

Comment:
Codacy found an issue: ['capitalize' is defined but never used.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6232875)